package.json

```diff
@@ -51,10 +51,12 @@
     "videojs-standard": "^6.0.0"
   },
   "optionalDependencies": {
+    "autoprefixer": "^8.6.0",
```

Comment:
another one to consider is ~cssnext~ postcss-preset-env (https://preset-env.cssdb.org/) (apparently, cssnext has been deprecated)